### PR TITLE
1368 fix pdf export templates

### PR DESCRIPTION
**Pull request by mki-c2c**
The 4 updates of V2 shall are put into the Jasper templates

- [x] GeneralInformation as an array
- [x] RealEstate.Municipality->RealEstate.MunicipalityName
- [x] RealEstate.MunicipalitFosNr->RealEstate.MunicipalityCode
- [x] UpdateDateOS->UpdateDateCS

### CODE REVIEW

**Comment by mki-c2c** [comment]
Multiple items in GeneralInformation renders as below:
before (single):
image
after (multiple items):
image

I cannot judge if multiple informations is a nominal or a special case. So we have to think if this formatting is OK.

**Comment by vvmruder** [comment]
This I always asked myself too. The Extract allows to have multiple elements. But on the page there is not really space for it as it is designed now?!

**Comment by voisardf** [comment]
Multiple informations is a case that can occur in many cantons. But it does not in all as far as I know.

File: print-apps/oereb/general_info_and_disclaimer.jrxml [hunk 1]
```diff
@@ -35,8 +35,8 @@
 				<textFieldExpression><![CDATA[$P{GeneralInformationLabel}]]></textFieldExpression>
 			</textField>
 		</band>
-	</title>
-	<pageHeader>
+	</detail>
+	<pageFooter>
```
Comment (marionb) [inline print-apps/oereb/general_info_and_disclaimer.jrxml:39]: this should work together with the change on line#64:
```suggestion

```

(The detail band is more flexible in size. The Footer is not resizable at run time. 
So in fact you could also adjust the "WYSIWYG" size of the bands and the text boxes to a minimum". But I leave that up to you if you have some spare time ;-) )

File: print-apps/oereb/general_info_and_disclaimer.jrxml [hunk 2]
```diff
@@ -61,5 +61,5 @@
 				<textFieldExpression><![CDATA[$P{DisclaimerLandRegister_Content}]]></textFieldExpression>
 			</textField>
 		</band>
-	</pageHeader>
+	</pageFooter>
```
Comment (marionb) [inline print-apps/oereb/general_info_and_disclaimer.jrxml:64]:
```suggestion
        </detail>
```
Maybe the spacing is not correct it is a bit hard with the git text editor.

**Review by marionb — APPROVED** [review]
LGTM thanks
Just one cosmetic improvement but it should also work without it

**Review by voisardf — APPROVED** [review]
thanks for the fixes and improvements. 👍

File: print-apps/oereb/general_info_and_disclaimer.jrxml (hunk 1 above)
Comment (mki-c2c) [reply print-apps/oereb/general_info_and_disclaimer.jrxml:39]: just tested that. But if I convert the footer into a detail element, it is repeated for each element of general information like this:
image
so I guess I'll keep it as is

**Pull request merged**